section3/reflection.md

```diff
@@ -2,12 +2,27 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+  It felt really good. I feel like I'm learning how to learn!
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+  I use the website Discogs everyday. Arrays are probably used when organizing and storing information about musical artists and releases.
+
 3. In your own words, explain what an Array method is.
 
+  Array methods are functionality built into JavaScript that allow you to work with elements within the array. You can add, subtract, reassign, sort ect.
+
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
 
+  A loop is used to check a condition a specific number of times or until a specific outcome is achieved.
+  You could use a loop with an array if you wanted to log the same text with different variables.
+  example " Hello <name>" with <name> being the variable run with however many names in an array.
+  You could us a loop on its own to run text a specific number of times or count up or down.
+
 5. If you had to teach a 5 year old about loops, how would you explain it?
 
+  A task is done over and over again until it is told to stop.
+
 6. What questions do you still have about Arrays and loops?
+
+  I'm still a little rocky with combining loops, arrays, and if/else all together. But practice will help.
```

Comment:
Exactly—like your reflection from section1 of the project, this is the start of putting all the pieces together. You're gonna be getting a ton of repetition in between class, homework, projects, studying—and it will all become second nature in a matter of weeks 🚀